tests/services/orders/order_store_service/test_microbial_fastq_order_store_service.py

```diff
@@ -0,0 +1,70 @@
+from datetime import datetime
+
+from cg.constants import Workflow, DataDelivery
+from cg.models.orders.order import OrderIn
+
+from cg.models.orders.constants import OrderType
+from cg.services.orders.store_order_services.store_microbial_fastq_order_service import (
+    StoreMicrobialFastqOrderService,
+)
+from cg.store.models import Sample, Case
+from cg.store.store import Store
+
+
+def test_microbial_samples_to_status(
```

Comment:
You could consider creating a fixture for how the expected dictionary should look like. Might make the test a bit cleaner. Seems to work as you want it to though 💪